frappe_telegram/override_doctype_class/notification.py

```diff
@@ -59,6 +59,22 @@ def send_telegram_notification(notification, doc):
             enqueue_after_commit=True
         )
 
+        if notification.attach_print:
```

Comment:
@Abadulrehman isn't the pdf-attachment same for all the users involved ?
won't it be better to make the attachment once, above the users-loop, and send the same for all ?

```suggestion
        if notification.attach_print:
            attachment = notification.get_attachment(doc)[0]
            ...
        
        for user in users:
            if not frappe.db.exists("Telegram User", {"user": user}):
                continue
            
            ...
```

Reply:
Makes sense. Fixed. 👍